### PR TITLE
W-038

### PR DESCRIPTION
**Pull request by MartynaSzymanskaGitHub**
refactoring Theme and font Size code

### CODE REVIEW

File: frontend/src/components/facility/ThemeContext.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useContext, useEffect, useState } from 'react';
+
+const ThemeContext = createContext();
+
```
Comment (wojciechwelfle) [inline frontend/src/components/facility/ThemeContext.jsx:4]: move context to new folder

File: frontend/src/pages/AdminPanelPage.jsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import AdminPanel from "../components/admin-panel/AdminPanel";
 import NavigationBar from "../components/navbar/Navbar";
+import { ThemeProvider } from '../components/facility/ThemeContext';
 
 const AdminPanelPage = () => {
     return (
         <>
+            <ThemeProvider>
```
Comment (wojciechwelfle) [inline frontend/src/pages/AdminPanelPage.jsx:8]: try use ThemeProvider in Rooting component

**Review by wojciechwelfle — CHANGES_REQUESTED** [review]

**Comment by MartynaSzymanskaGitHub** [comment]
changed files

**Comment by MartynaSzymanskaGitHub** [comment]
Fix text position

**Review by wojciechwelfle — APPROVED** [review]
lgtm

**Pull request merged**